Dockerfile

```diff
@@ -65,6 +65,7 @@ RUN echo "import sys\nsys.path.append('/source/python')" > /etc/python2.7/sitecu
 
 ADD tools/docker/supervisord.conf /etc/supervisor/conf.d/nav.conf
 
+COPY requirements/ /requirements
```

Comment:
Bare en liten kommentar:

Har ikke så mye peil på Dockerfiles, men er ikke COPY bare et subsett av ADD? Og dermed, for å være konsekvent, burde man bruke en av de?